obspy/io/gse2/libgse2.py

```diff
@@ -465,13 +465,33 @@ def parse_sta2(line):
     """
     header = {}
     try:
+        lat = line[15:24].strip()
```

Comment:
Yeah, that was my line of thought. I don't want to turn invalid entries into `None`, just blank entries.

In a quick search, couldn't find what GSE manual says about blank entries. Real world files seem to have invalid values -- like `-999.00` for the latitude -- in case of missing values, to the extent of breaking the fixed format even some times. I think for a fixed format blanks make the most sense, so I decided to go with that -- in writing and thus obviously in reading, too.